packages/react-native/prepare-android-vendor.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+TEMP_DIR=
+cleanup() {
+    if [ "$TEMP_DIR" != "" ]; then
+        rm -rf "$TEMP_DIR"
+    fi
+}
+trap cleanup EXIT
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+SCRIPT_NAME="${0##*/}"
+CONFIG_FILE="$SCRIPT_DIR/prepare-android-vendor.config"
+
+MODE=$(head -1 "$CONFIG_FILE")
+ARG=$(tail -1 "$CONFIG_FILE")
+
+MAVEN_REPO_DIR="$HOME/.m2/repository/com/bugsnag"
```

Comment:
The intent behind this change is that the artefacts are downloaded from [maven central](https://search.maven.org/) so that it's not necessary to build them twice, which could potentially result in different binaries if done on different machines. It doesn't look like this would achieve that requirement as it's building the artefact locally.